Control coverage/Feature/CDN.md

```diff
@@ -62,3 +62,52 @@ One of the following conditions is met:
 properties.isHttpAllowed, properties.isHttpsAllowed, properties.deliveryPolicy.rules
  <br />
 
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial 
+
+### Display Name 
+[Trial] Front Door should have Approved Minimum TLS version
+
+### Rationale 
+TLS provides privacy and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions. 
+
+### Control Spec 
+
+> **Passed:** 
+>- For all custom domain, the Minimun TLS Version is set to TLS 1.2 or greater than that.
+> 
+> **Failed:** 
+>- Custom domain Minimum TLS Version is less than TLS 1.2 for any domain. 
+>
+
+### Recommendation 
+
+- **Azure Portal** 
+    To configure TLS Version, Go to Azure Portal --> Front Door and CDN profiles -> Select Front door with pricing tier as Standard/Premium -> Goto Domains -> Select Certification Type for All the domains listed -> Select Minimum TLS Version as 1.2
+
+- **PowerShell** 
+
+	 
+	 ```powershell
+	$secret =  Get-AzFrontDoorCdnSecret -ResourceGroupName $resourceGroupName -ProfileName $resourceName
+	$secretResoure = New-AzFrontDoorCdnResourceReferenceObject -Id $secret.Id
+    $updateTlsSetting = New-AzFrontDoorCdnCustomDomainTlsSettingParametersObject -CertificateType $CertificateType -MinimumTlsVersion 'TLS12' -Secret $secretResoure
```

Comment:
Can we provide details about how to get '$CertificateType'

Reply:
Added API call for getting certificate type.